drv/fe310-rtc/src/main.rs

```diff
@@ -5,19 +5,28 @@
 #![no_std]
 #![no_main]
 
+use drv_riscv_plic_api::*;
 use ringbuf::*;
 #[allow(unused_imports)]
 use userlib::*;
 
+task_slot!(INT_CONTROLLER, ext_int_ctrl);
+
 #[export_name = "main"]
 fn main() -> ! {
     const RTC_INT: u32 = 0x1;
```

Comment:
Would be nice if this was automatically exported based on the app.toml

Reply:
The name of the task slot (`INT_CONTROLLER` here) is only defined within this file, so it's not feasible for now. But if there was some way to get it, this could be added in along with the const generation for peripheral addresses and notification masks.

Reply:
Missed the line with this one.  Should have been on 17. `const RTC_INT: u32 = 0x1;`

Reply:
I'll be adding that in a later patch.